x-pack/plugins/security_solution/common/entity_analytics/risk_engine/risk_levels.ts

```diff
@@ -5,26 +5,26 @@
  * 2.0.
  */
 
-import { RiskLevels } from './types';
+import { EntityRiskLevels } from '../../api/entity_analytics/common';
 
 export const RISK_LEVEL_RANGES = {
-  [RiskLevels.unknown]: { start: 0, stop: 20 },
-  [RiskLevels.low]: { start: 20, stop: 40 },
-  [RiskLevels.moderate]: { start: 40, stop: 70 },
-  [RiskLevels.high]: { start: 70, stop: 90 },
-  [RiskLevels.critical]: { start: 90, stop: 100 },
+  [EntityRiskLevels.enum.Unknown]: { start: 0, stop: 20 },
```

Comment:
nit: we could use EntityRiskLevelsEnum type here

Reply:
I didn't realise that it also generated enums. I will use them. Thank you!